_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -15166,21 +15166,8 @@
 /area/maintenance/disposal)
 "aNV" = (
 /obj/machinery/conveyor{
-	dir = 4;
-	id = "garbagestacked"
-	},
-/obj/machinery/mineral/stacking_unit_console{
```

Comment:
Did you accidentally replace all objects on this tile instead of inserting?

Reply:
actually looks like it 2 seconds.

Reply:
fixed